src/Mapping/Event/AdapterInterface.php

```diff
@@ -5,12 +5,11 @@
 use Doctrine\Common\EventArgs;
 use Doctrine\ORM\UnitOfWork;
```

Comment:
IMO, if this class is not used, we should remove this import.

Reply:
It's used.